azure-pipelines.yml

```diff
@@ -11,6 +11,8 @@ trigger:
       - NEWS
       - UPGRADING
       - UPGRADING.INTERNALS
+      - *.md
+      - ext/*
```

Comment:
This does not look correct, as most extension code lives here

Reply:
the solution will be <code>ext/*.md</code>